### PR TITLE
Update set-variable.mdx

### PR DESCRIPTION
**Pull request by CptPlastic**
Added location error handling.

## Summary by CodeRabbit

- **Refactor**
	- Updated location retrieval functionality:
		- Renamed function for clarity.
		- Enhanced location information to include latitude and longitude.
		- Improved error handling for smoother user experience.

### CODE REVIEW

**Comment by CptPlastic** [comment]
Probably add a note that says to run the block as "client" some people may not know to do that for the clients browsers.

**Comment by baptisteArno** [comment]
> Probably add a note that says to run the block as "client" some people may not know to do that for the clients browsers.

Indeed, would be nice! Any chance you can add that as well 🙏

File: apps/docs/editor/blocks/logic/set-variable.mdx [hunk 1]
```diff
@@ -127,17 +127,23 @@ This value block allows you to find the `Id` from `Ids` with the same index as `
...
 
-return `${coords.latitude}, ${coords.longitude}`
+// Check for error
+if (coords === "error") {
+  return "Unable to get location";
+}
+
+return coords;
 ```
+Make sure to select "Exacute on client" is selected.
```
Comment (baptisteArno) [inline apps/docs/editor/blocks/logic/set-variable.mdx:149]:
```suggestion

This custom function can only work when it is executed on the client browser so you need to make sure to enable the "Execute on client" option.
```
Make sure to select "Exacute on client" is selected.
```

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Just a tiny improv! 🙏

Thank you my friend

**Review by baptisteArno — APPROVED** [review]

**Pull request merged**